TwitchPlaysAssembly/Src/Commands/GlobalCommands.cs

```diff
@@ -647,7 +646,7 @@ public static void RunHelp()
 				return null;
 			}
 
-			if (!Leaderboard.Instance.IsAnyEvil())
+/* 			if (!Leaderboard.Instance.IsAnyEvil())
```

Comment:
please ensure that there are both evil and good players before starting a VS bomb, it's very unbalanced if there aren't

Reply:
I had actually forgotten I had commented that out in the process of trying to figure out how to get it to start. :p